vignettes/xcms.Rmd

```diff
@@ -431,6 +431,56 @@ numeric).
 chromPeakData(faahko)
 ```
 
+### Chromatographic peak quality
+
+Based on the publication by Kumler et al. published in 2023 [@Kumler2023], new
+quality metrics (*beta_cor* and *beta_snr*) were added to *xcms*. *beta_cor*
+indicates how bell-shaped the chromatographic peak is and *beta_snr* is
+estimating the signal-to-noise ratio using the residuals from this fit. These
```

Comment:
Could be worth mentioning here why this metric might differ from the existing "sn" column. In my experience this mostly occurs with chromatographic peaks that don't have very long tails and there isn't enough information outside of the peak to robustly estimate noise, resulting in a hard-coded value of noise = 1. So that people could still use the sn metric if they know to be skeptical of the times when sn = floor(maxo).